docs/tutorials/analysis/3D/event_sampling.ipynb

```diff
@@ -77,6 +77,8 @@
     "from pathlib import Path\n",
```

Comment:
<p>You could create a lightcurve out of the simulated events and plot it together with the temporal model to illustrate the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>